mix.exs

```diff
@@ -59,15 +59,17 @@ defmodule ExMicrosoftbot.Mixfile do
 
   defp deps do
     [
-      {:httpotion, "~> 3.0.0"},
+      {:httpoison, "~> 1.7"},
       {:poison, "~> 4.0"},
       {:jose, "~> 1.7"},
       {:timex, "~> 3.0"},
       {:tzdata, "~> 1.0"},
       {:inch_ex, "~> 2.0.0", only: :docs},
       {:dialyxir, "~> 0.3", only: [:dev]},
       {:ex_doc, "~> 0.19", only: [:dev]},
-      {:bypass, "~> 1.0", only: :test}
+      {:bypass, "~> 1.0", only: :test},
+      # Required by bypass, incompatible with OTP 22 since 2.8.0:
```

Comment:
The lib supports Elixir from 1.8, which is [compatible only with OTP 20-21](https://hexdocs.pm/elixir/1.8.2/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp).